PlayCover/Views/AppSettingsView.swift

```diff
@@ -477,22 +484,36 @@ struct BypassesView: View {
                 HStack {
                     Toggle("settings.toggle.introspection", isOn: $hasIntrospection)
                         .help("settings.toggle.introspection.help")
+                        .toggleStyle(.async($task, role: .introspection))
                     Spacer()
                 }
                 Spacer()
                 HStack {
                     Toggle("settings.toggle.iosFrameworks", isOn: $hasIosFrameworks)
                         .help("settings.toggle.iosFrameworks.help")
+                        .toggleStyle(.async($task, role: .iosFrameworks))
                     Spacer()
                 }
             }
             .padding()
         }
         .onChange(of: hasIntrospection) {_ in
-            _ = app.changeDyldLibraryPath(set: hasIntrospection, path: PlayApp.introspection)
+            task = .introspection
+            Task {
+                _ = await app.changeDyldLibraryPath(set: hasIntrospection, path: PlayApp.introspection)
+                task = .none
+            }
         }
         .onChange(of: hasIosFrameworks) {_ in
-            _ = app.changeDyldLibraryPath(set: hasIosFrameworks, path: PlayApp.iosFrameworks)
+            task = .iosFrameworks
+            Task {
+                _ = await app.changeDyldLibraryPath(set: hasIosFrameworks, path: PlayApp.iosFrameworks)
+                task = .none
+            }
+        }
+        .task {
```

Comment:
Loading these variables should be kept passed in, as this is not a blocking task. Another thing with having the variable being set here is that it triggers the `onChange` event, which causes the loading to take unnecessarily longer.